src/otx/core/data/dataset/base.py

```diff
@@ -83,6 +83,7 @@ def __init__(
         image_color_channel: ImageColorChannel = ImageColorChannel.RGB,
         stack_images: bool = True,
         to_tv_image: bool = True,
+        data_format: str = "",
```

Comment:
Could you add a docstring for `data_format`?

Reply:
No problem, will add it when starting finalization of the PR

Reply:
Done